addons/l10n_in_edi/models/account_edi_format.py

```diff
@@ -196,7 +197,7 @@ def _cancel_invoice_edi(self, invoices):
                         "error": self._l10n_in_edi_get_iap_buy_credits_message(invoice.company_id),
                         "blocking_level": "error",
                     }
-                else:
+                if error:
```

Comment:
better to not put an empty attachment in the case of code 9999

Reply:
Hello  @jco-odoo,
I have done the changes, can you please check it ?